### PR TITLE
feat: Badge Type explorer with info preview

### PR DESCRIPTION
**Pull request by FedeMadoery**
# Description

- New Badge Explorer with preview information about the strategy/type
- New Badge Explorer where you can see all the minted badges for a given type/strategy (For testing now, not a feature yet)
- Fix useS3Metadta to cache the results and prevent 429 Error with our API
- Display User Info (Avatar/Name/Description/Contact) on Profile page

### CODE REVIEW

File: pages/badge/explorer.tsx [hunk 1]
```diff
@@ -48,58 +73,80 @@ const ExploreBadgeTypes: NextPageWithLayout = () => {
...
         categories={['Category 1', 'Category 2', 'Category 3']}
         filters={filters}
         loading={loading}
         loadingColor={'blue'}
+        preview={
+          badgeTypes[selectedBadgeType] && (
+            <SafeSuspense>
+              <Box display="flex" justifyContent="space-between">
+                <Typography color={colors.blue} mb={4} variant="dAppHeadline2">
+                  BadgeType Info
```
Comment (ramabit) [inline pages/badge/explorer.tsx:110]: please move text to translations file

File: pages/badge/explorer.tsx [hunk 2]
```diff
@@ -48,58 +73,80 @@ const ExploreBadgeTypes: NextPageWithLayout = () => {
...
+
   return (
     <>
       <FilteredList
         categories={['Category 1', 'Category 2', 'Category 3']}
         filters={filters}
         loading={loading}
         loadingColor={'blue'}
+        preview={
+          badgeTypes[selectedBadgeType] && (
```
Comment (ramabit) [inline pages/badge/explorer.tsx:106]: perhaps this preview can be delegated to a function, so the code looks cleaner and easier to understand

File: pages/badge/explorer.tsx [hunk 3]
```diff
@@ -48,58 +73,80 @@ const ExploreBadgeTypes: NextPageWithLayout = () => {
...
+                  onClick={() => setSelectedBadgeType(i)}
+                  selected={bt.id === badgeTypes[selectedBadgeType]?.id}
+                >
+                  <MiniBadgeTypeMetadata
+                    buttonTitle={t('explorer.button')}
+                    disableAnimations
+                    highlightColor={colors.blue}
+                    metadata={bt.metadataURL}
+                    onClick={() => {
+                      // router.push(`/badge/mint/${bt.id}`)
```
Comment (ramabit) [inline pages/badge/explorer.tsx:143]: no behavior on click? need a 'TODO' here?

File: src/components/helpers/FilteredList.tsx [hunk 4]
```diff
@@ -35,13 +35,13 @@ type FilteredListProps = PropsWithChildren & {
...
   loadingColor?: SpinnerColors
   disableEdit?: boolean
+  preview?: ReactNode | undefined
 }
 
 const ItemsGridBox = styled(Box)(({ theme }) => ({
   display: 'flex',
   flexWrap: 'wrap',
   gap: theme.spacing(3),
-  justifyContent: 'center',
```
Comment (ramabit) [inline src/components/helpers/FilteredList.tsx:44]: how affects different viewports if we remove the centering?

Comment (FedeMadoery) [reply src/components/helpers/FilteredList.tsx:44]: It has been moved to the usage, bc its changes based on the preview element

File: i18n/translations.en.json [hunk 5]
```diff
@@ -53,7 +53,25 @@
...
+      "badge": {
+        "name": "Name",
+        "description": "Description",
+        "mintCost": "Mint cost:",
+        "totalMinted": "Total minted amount:",
+        "mint": "Mint",
+        "showOthers": "See minted badges"
+      },
+      "creator": {
+        "createdBy": "Crated By"
```
Comment (ramabit) [inline i18n/translations.en.json:68]: Created By*

**Review by ramabit — CHANGES_REQUESTED** [review]

**Pull request merged**